### PR TITLE
DRAFT: Translucency sorting

### PR DESCRIPTION
**Pull request by MCRcortex**
Draft GPU driven translucency sorting

### CODE REVIEW

File: src/main/java/me/jellysquid/mods/sodium/client/SodiumPreLaunch.java [hunk 1]
```diff
@@ -16,6 +16,7 @@ public class SodiumPreLaunch implements PreLaunchEntrypoint {
 
     @Override
     public void onPreLaunch() {
+        System.load("C:\\Program Files\\RenderDoc\\renderdoc.dll");
```
Comment (MeeniMc) [inline src/main/java/me/jellysquid/mods/sodium/client/SodiumPreLaunch.java:19]: you probably don't wnat to PR this?

**Comment by NathanaelLane** [comment]
As mentioned in [this reddit thread](https://old.reddit.com/r/VoxelGameDev/comments/a0l8zc/correct_depthordering_for_translucent_discrete/), render order follows a planar generalization of the classic DDA algorithm, which means that with some clever bookkeeping it's not technically necessary to do a full sort-- this could possibly be exploited to exchange the BTMS for a straightforward shuffle routine. (Well, assuming you haven't considered this already :slightly_smiling_face: )

**Comment by douira** [comment]
We have to take into account that the translucent quads we're sorting aren't necessarily axis-aligned or part of complete voxels. Water being the prime example.

**Comment by douira** [comment]
Relevant to the translucency sorting endeavor and this PR in particular as it's in need of a triggering mechanism: I wrote a whole bunch about a new plane-crossing based translucency sorting triggering mechanism in this document https://hackmd.io/@douira100/sodium-sl-gfni

**Comment by nonetrix** [comment]
Was this fixed???

**Comment by hexadecimal233** [comment]
just closed ig

**Pull request closed without merging**